wandb/apis/public.py

```diff
@@ -960,7 +960,8 @@ def _sampled_history(self, keys, x_axis="_step", samples=500):
         ''')
 
         response = self._exec(query, specs=[json.dumps(spec)])
-        return [line for line in response['project']['run']['sampledHistory']]
+        # sampledHistory returns one list per spec, we only send one spec
+        return response['project']['run']['sampledHistory'][0]
```

Comment:
@raubitsj @shawnlewis I snuck this one in for Onfido.  This will break existing users if they adjusted for this, but it should be rare.  It was hidden because when we return a dataframe it flattens the array, this would only effect users that called `run.history(key=[...], pandas=False)`, instead of returning a 2d array we just return a 1d array.

Reply:
stealthy fix, i like it